src/components/Purchase/CheckoutPage.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import { loadStripe } from '@stripe/stripe-js';
+import { Elements } from '@stripe/react-stripe-js';
+import style from './CheckoutPage.module.css';
+import CheckoutForm from './CheckoutForm';
+
+// Make sure to call loadStripe outside of a component’s render to avoid
+// recreating the Stripe object on every render.
+// This is your test publishable API key.
+const stripePromise = loadStripe(
+  'pk_test_51Nfj2pKnb0YvaiB3W0z31bbB7OPEnqem6wnpSbohudeiDkj1NUde9M5kgUwozzrfBqGqRpWU5ivDIInzWdt4q5zw00vMz7fc1c'
+);
+const paymentAmount = 1099; // This is $10.99 payment amount(added for testing) This amount variable should came from cart
+
+const CheckoutPage = () => {
+  const [clientSecret, setClientSecret] = useState('');
+
+  useEffect(() => {
+    // Create PaymentIntent as soon as the page loads
+    fetch('http://localhost:8000/api/v1/stripe', {
```

Comment:
As well, that endpoint requires a cookie because the backend needs to authenticate who the user is who's making the order and that they've logged in. 

With `fetch` you can send cookies by adding `credentials: 'include'` to your options: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#sending_a_request_with_credentials_included

If you don't mind though I'd ask if you can convert this from using `fetch` to using the axios client defined in the `src/apis` folder to follow the pattern/standard that we'll be trying to use in this repo. More info here: https://codethedreamteam4-2023.atlassian.net/wiki/spaces/PT4/pages/5734401/Making+API+calls+from+the+frontend

Reply:
I did changed the endpoint to http://localhost:8000/api/v1/orders, thanks.
And thank you for pointing me to Cookies issue. Which exact Cookies I should include in my callout?
Sure, I will try to convert fetch to axios after I will resolve authorizations issues.

Reply:
I have included credentials: 'include' to my API call.

Reply:
I added order info to my API request, but the code still doesn't work. What other changes do I need to make? Can you please advise something?

Reply:
The problem resolved. The issue was related to variable name in .env